### ISSUE TITLE
❓Request for Invoice for License Purchase Dated February 19, 2024

### ISSUE THREAD

**Issue by AndriiVavrynets** [inovua/reactdatagrid#419]
Dear React Data Grid Support Team,

I hope this message finds you well. I am reaching out to request the invoice for a license we purchased through your website on February 19, 2024. Despite several attempts to contact you over the past months, we have yet to receive the corresponding invoice.

It is crucial for us to have the official invoice for our accounting records. Could you please provide us with the invoice at your earliest convenience? If you require any additional information to locate our transaction or to issue the invoice, please feel free to contact me.

Thank you for your attention to this matter. I look forward to your prompt response.

**Comment by edy** [comment]
@AndriiVavrynets same here, did you get your invoice?

**Comment by AndriiVavrynets** [comment]
Nope. I got nothing from React Data Grid Team, no email, no message, no invoice.

Best regards
Andrii
